### DISCUSSION TITLE
How do I hide and show preview?

### DISCUSSION THREAD

**Discussion by GZLiew** [PatrickF1/fzf.fish#266] [Q&A] [answered]
How do I hide and show the preview window?

**Comment by PatrickF1** [comment, marked as answer]
Are you asking how to hide and show preview while already in fzf? For that, you'll need to pass in a keybinding to call the `toggle-preview` action. e.g. `set fzf_dir_opts --bind ctrl-h:toggle-preview` will allow you to toggle the preview using Ctrl+h for the search files feature.

If you just want to disable it completely, then pass in `--preview=""`.

Edit: just added this to the cookbook https://github.com/PatrickF1/fzf.fish/wiki/Cookbook#how-can-i-toggle-the-preview-window-on-and-off-while-in-fzf.

**Reply by GZLiew** [reply]
ah cool that worked yup I was searching around the repo turns out is at the end of the WIKI link. Thanks @PatrickF1 !

**Reply by PatrickF1** [reply]
Oh that's cause I added it there after I answered your question :)
Thanks for your great question. I'm sure others will have the same question as well.
